seirmo/model.py

```diff
@@ -0,0 +1,37 @@
+#
+# ForwardModel Class
+#
+
+
+class ForwardModel:
```

Comment:
Could you please specify that ForwardModel is inheriting from the `object` class, i.e.
```
class ForwardModel(object):
   ...
```

By default FowardModel will inherit from object, but it's nicer to make it explicit.

Reply:
Done